src/SSHLibrary/library.py

```diff
@@ -901,13 +901,22 @@ def login(self, username, password, allow_agent=False, look_for_keys=False, dela
         | `Open Connection` | linux.server.com | prompt=$         |
         | ${output}=        | `Login`          | johndoe          | secretpasswd |
         | `Should Contain`  | ${output}        | johndoe@linux:~$ |
+
+        Example that logs in through a proxy server (10.128.3.101)
```

Comment:
Instead of `10.128.3.101` you could use `proxy.server.com` to be in line with previous examples.